lib/lf/uscalfe/mesh_function_binary.h

```diff
@@ -9,7 +9,7 @@
 #ifndef __9bad469d38e04e8ab67391ce50c2c480
 #define __9bad469d38e04e8ab67391ce50c2c480
 
-#include "lf/mesh/mesh_interface.h"
+#include <unsupported/Eigen/KroneckerProduct>
```

Comment:
Why the Kronecker product? Isn't this a bit too special?

Reply:
Ah thanks for pointing this out. At first I wanted to use the kronecker product for one of the implementations but then I figured out a better way. I will remove the include here :)